third_party/move/move-compiler/src/parser/syntax.rs

```diff
@@ -96,6 +96,29 @@ fn require_move_2(context: &mut Context, loc: Loc, description: &str) -> bool {
     }
 }
 
+fn require_language_version(
+    context: &mut Context,
+    loc: Loc,
+    min_language_version: LanguageVersion,
+    description: &str,
+) -> bool {
+    if context.env.flags().language_version() < min_language_version {
+        context.env.add_diag(diag!(
+            Syntax::UnsupportedLanguageItem,
+            (
+                loc,
+                format!(
+                    "Move {} language construct is not enabled: {}",
```

Comment:
This would probably be better as
```
"Move {} language construct `{}` is not enabled"
```
And maybe we could tell how to enable that version, if we have a standard way.

Reply:
Or maybe:
```
"Move language construct `{}` is not enabled until version {}", description, min_language_varsion
```

Reply:
done